cmd/sweepaccount/main.go

```diff
@@ -277,10 +274,14 @@ func sweep() error {
 		numErrors++
 	}
 	for _, previousOutputs := range sourceOutputs {
-		inputSource := makeInputSource(previousOutputs)
+		inputs, err := fetchInputs(previousOutputs)
+		if err != nil {
+			return err
+		}
 		destinationSource := makeDestinationScriptSource(rpcClient, opts.DestinationAccount)
+		// We are only selecting postive yieling outputs.
 		tx, err := txauthor.NewUnsignedTransaction(nil, opts.FeeRate.Amount,
```

Comment:
Did you test the sweep action to see if you have any regression? (as there is no test for it)

Reply:
good point